gprofiler/main.py

```diff
@@ -602,6 +605,30 @@ def parse_cmd_args() -> configargparse.Namespace:
         " beginning of a session.",
     )
 
+    parser.add_argument(
+        "--limit-memory",
+        default=(1 << 30),  # 1Gi, same as in the k8s DaemonSet
+        dest="memory_limit",
+        type=int,
+        help="Limit on the memory used by gProfiler."
+    )
+
+    parser.add_argument(
+        "--limit-cpu",
+        default=0.5,  # 500m, same as in the k8s DaemonSet
+        dest="cpu_limit",
+        type=float,
+        help="Limit on the cpu used by gProfiler."
+    )
+
+    parser.add_argument(
+        "--no-cgroups",
+        action="store_true",
+        dest="disable_cgroups",
```

Comment:
```suggestion
        dest="cgroups_changes",
```
Code is more clear if it works on the "positive" case and not the "negative" case. The argument can then be `store_false`.